src/internal/calculateTransform.js

```diff
@@ -26,32 +26,36 @@ export default function (enabledElement, scale) {
   let widthScale = enabledElement.viewport.scale;
   let heightScale = enabledElement.viewport.scale;
 
-  const width = enabledElement.viewport.displayedArea.brhc.x - (enabledElement.viewport.displayedArea.tlhc.x - 1);
-  const height = enabledElement.viewport.displayedArea.brhc.y - (enabledElement.viewport.displayedArea.tlhc.y - 1);
-
-  if (enabledElement.viewport.displayedArea.presentationSizeMode === 'NONE') {
+  const { displayedArea } = enabledElement.viewport;
+  const { tlhc, brhc, presentationSizeMode } = displayedArea;
+  const offsetX = tlhc.x - 1;
+  const offsetY = tlhc.y - 1;
```

Comment:
why these are called offset?

Reply:
I think the top left-hand corner (tlhc) basically means an offset

Reply:
it has been defined elsewhere as sx and sy